Classes/KIFUITestActor.m

```diff
@@ -444,7 +444,7 @@ - (void)expectView:(UIView *)view toContainText:(NSString *)expectedResult
         KIFTestWaitCondition([actual isEqualToString:expected], error, @"Failed to get text \"%@\" in field; instead, it was \"%@\"", expected, actual);
         
         return KIFTestStepResultSuccess;
-    } timeout:1.0];
+    } timeout:2.0];
```

Comment:
I agree. I don't see why not just use the default 10s timeout here if a failure is going to get raised on timeout anyway.